plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -13,24 +13,26 @@
 ---
 module: newrelic_deployment
 author: "Matt Coddington (@mcodd)"
-short_description: Notify newrelic about app deployments
+short_description: Notify New Relic about app deployments
 description:
-   - Notify newrelic about app deployments (see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/deployment-notifications#api)
+  - Notify New Relic about app deployments (see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/record-monitor-deployments/)
 options:
   token:
     type: str
     description:
-      - API token, to place in the x-api-key header.
+      - API token, to place in the Api-Key header.
     required: true
   app_name:
     type: str
     description:
-      - (one of app_name or application_id are required) The value of app_name in the newrelic.yml file used by the application
+      - The value of app_name in the newrelic.yml file used by the application.
+      - One of I(app_name) or I(application_id) is required.
     required: false
   application_id:
     type: str
     description:
-      - (one of app_name or application_id are required) The application id, found in the URL when viewing the application in RPM
+      - The application ID, found in the metadata of the application in APM.
```

Comment:
Nit picking on the commas
```suggestion
      - The application ID found in the metadata of the application in APM.
```